### ISSUE TITLE
retries and interval are not working

### ISSUE THREAD

**Issue by jaydeep-sh** [component-driven/cypress-axe#184]
When trying following code, retries and thus interval does not happen when a11y violations are reported.

cy.checkA11y(
    null,
    {
      retries: 3,
      interval: 1000
},
null,
        true,
      )

**Comment by adam-dunbabin** [comment]
I am finding the same problem.  Using cypress-axe 1.7.0 with cypress 13.5.1.

Can someone please take a look at this as it could be a really useful feature - if it worked!
